cmd/skopeo/sync.go

```diff
@@ -578,7 +580,7 @@ func (opts *syncOptions) run(args []string, stdout io.Writer) error {
 		OptimizeDestinationImageAlreadyExists: true,
 		ForceManifestMIMEType:                 manifestType,
 	}
-
+	errorsPresent := false
```

Comment:
Non-blocking: the `imagesNumber` and `errorsPresent` values have the same lifetime and conceptual purpose, it would be nice for them to be initialized/updated/used together.

I.e., in this case, move the `imagesNumber := 0` initialization above just next to this newly added line.

Reply:
Done in latest update to this PR.